pkgs/top-level/all-packages.nix

```diff
@@ -17687,6 +17687,7 @@ in
   wavpack = callPackage ../development/libraries/wavpack { };
 
   wayland = callPackage ../development/libraries/wayland { };
+  wayland-scanner = wayland.bin;
```

Comment:
This is optional btw. I think that alias makes sense but apparently only Gentoo packages it that way: https://repology.org/project/wayland-scanner/versions

We could also drop this part and continue adding `wayland` to `nativeBuildInputs` or even create an independent `wayland-scanner` package that only builds the binary and let `wayland` only build the libs (but this should break a lot of existing packages...).

Reply:
I think this is fine and it should make it easier to port packages to darwin.